### DISCUSSION TITLE
Usecases for Event.FRAME_CONSTRUCTED?

### DISCUSSION THREAD

**Discussion by ylazy** [airsdk/Adobe-Runtime-Support#2741] [General]
Hi!

Many years playing with ActionScript but I have not ever used the `Event.FRAME_CONSTRUCTED`.

I can't find any usecase for this.

Any ideas?

Thanks!

**Comment by henke37** [comment]
It is used when working with timeline animations. Remember the event order:

1. Enter frame
2. Frame scripts
3. Frame constructed
4. Interactive wait period. (All async events happen here, Wait set to zero if lagging)
5. Exit frame
6. Render (Skipped if lagging)
